tests/form_autofill/test_autofill_credit_card.py

```diff
@@ -0,0 +1,44 @@
+from selenium.webdriver import Firefox
+
+from modules.browser_object import CreditCardPopup, Navigation
+from modules.browser_object_autofill_popup import AutofillPopup
+from modules.page_object_autofill_credit_card import CreditCardFill
+from modules.util import Utilities
+
+
+def test_autofill_credit_card(driver: Firefox):
+    """
+    C122405, tests that after filling autofill and disabling cc info it appears in panel
+    """
+    nav = Navigation(driver)
+    util = Utilities()
+
+    nav.open()
+    ccf = CreditCardFill(driver).open()
```

Comment:
I would prefer longer names for these variables. If this file grows it may become difficult to remember what each of these acronyms refers to.